src/paperless/settings.py

```diff
@@ -329,6 +329,7 @@ def _parse_beat_schedule() -> dict:
     "allauth.mfa",
     "drf_spectacular",
     "drf_spectacular_sidecar",
+    "cachalot",
```

Comment:
Can we include this only conditionally?

I see a number of [checks](https://github.com/noripyt/django-cachalot/blob/master/cachalot/apps.py) I think this will add otherwise, for people who have not opted in to the feature